sdk/communication/Azure.Communication.Chat/src/ChatClient.cs

```diff
@@ -62,7 +62,17 @@ public virtual async Task<Response<CreateChatThreadResult>> CreateChatThreadAsyn
             scope.Start();
             try
             {
-                Response<CreateChatThreadResultInternal> createChatThreadResultInternal = await _chatRestClient.CreateChatThreadAsync(topic, idempotencyToken, participants.Select(x => x.ToChatParticipantInternal()), null, cancellationToken).ConfigureAwait(false);
+                var createChatThreadOptions = new CreateChatThreadOptions(topic)
+                {
+                    IdempotencyToken = idempotencyToken,
+                };
+
+                foreach (var value in participants.ToList())
+                {
+                    createChatThreadOptions.Participants.Add(value);
+                }
+
```

Comment:
Why do we need changes here? Seems the `ToChatParticipantInternal()` is removed from the function.

Reply:
Oh, I see the refactor in the ChatRestClient